### DISCUSSION TITLE
Icon request: phases of the moon (8 icons)

### DISCUSSION THREAD

**Discussion by mattnraines** [FortAwesome/Font-Awesome#21137] [Icon Requests]
A project I'm working on needs icons for the 8 standard [phases of the moon](https://simple.wikipedia.org/wiki/Phases_of_the_Moon).

Any chance fa-moon-new, fa-moon-waxing-crescent, fa-moon-first-quarter, fa-moon-waxing-gibbous, fa-moon-full, fa-moon-waning-gibbous, fa-moon-last-quarter, and fa-moon-waning-crescent could make it into the icon set?

In the meantime I've made do with far fa-circle, fas fa-circle, fa-adjust, masking fa-circle with fa-moon, and various combinations of flip and resize but my attempts at gibbous, in particular, don't look amazing.

**Comment by duckrising** [comment]
+1

**Comment by c64emulator** [comment]
+1

**Comment by 5orenso** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🌑 [U+1F311](https://emojipedia.org/emoji/%F0%9F%8C%91/) New Moon
🌒 [U+1F312](https://emojipedia.org/emoji/%F0%9F%8C%92/) Waxing Crescent Moon
🌓 [U+1F313](https://emojipedia.org/emoji/%F0%9F%8C%93/) First Quarter Moon
🌘 [U+1F318](https://emojipedia.org/emoji/%F0%9F%8C%98/) Waning Crescent Moon
🌕 [U+1F315](https://emojipedia.org/emoji/%F0%9F%8C%95/) Full Moon
🌖 [U+1F316](https://emojipedia.org/emoji/%F0%9F%8C%96/) Waning Gibbous Moon
🌗 [U+1F317](https://emojipedia.org/emoji/%F0%9F%8C%97/) Last Quarter Moon
🌘 [U+1F318](https://emojipedia.org/emoji/%F0%9F%8C%98/) Waning Crescent Moon

@tagliala `weather (category)` label for this request?

**Comment by lithxe** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by jaworskimatt** [comment]
I know it's been around a while, but a +1 from me. Apple SF already has these, but I'm building an iOS + Android app, and for Android we use FA for icons

<img width="3490" height="2274" alt="Image" src="https://github.com/user-attachments/assets/ef4e986e-b2fa-401a-890d-fe53dec0ea08" />

**Comment by sensibleworld** [comment]
We've added moon phase icons to 7.2: https://fontawesome.com/search?q=moon%20phase&ip=classic&s=solid
